apps/challenges/views.py

```diff
@@ -132,3 +133,17 @@ def disable_challenge(request, pk):
         response_data = {
             'error': 'Sorry, you do not have permission to disable this challenge'}
         return Response(response_data, status=status.HTTP_401_UNAUTHORIZED)
+
+
+@api_view(['GET'])
+def get_past_challenges(request):
```

Comment:
I think the challenge details should be visible to everyone so that we can index these pages. Does that make sense?